### ISSUE TITLE
Difficult to recognize Chinese and English symbols: {}

### ISSUE THREAD

**Issue by SunBrook** [JetBrains/JetBrainsMono#641]
Chinese symbols: `｛｝`
English symbols: `{}`

![image](https://github.com/JetBrains/JetBrainsMono/assets/14243713/ceea65cc-6dc4-4959-8ad1-4acf43594d40)

**Comment by YVVT** [comment]
After testing, I found that the fullwidth curly braces `｛｝` are the same width as other chars. JetBrains Mono should not have these symbols.

![image](https://github.com/JetBrains/JetBrainsMono/assets/62426149/88ce410c-7136-45db-a895-60bd96036db5)

**Comment by GitXiaoJie** [comment]
Yes, this causes an error when I use log.info("{}"), it actually is log.info("｛｝")
